### ISSUE TITLE
failed to execute tmux send-keys -t

### ISSUE THREAD

**Issue by rif** [fzerorubigd/tmass#7]
I have the following setup:
- linux (arch)
- zsh (5.0.7)
- tmux (1.9a)

❯ pwd
/home/rif/.config/tmass
❯ cat two-window-per-session.yml 
catname: two-window-per-session
windows:
  - name: window-1
    layout: main-vertical
    panes:
      - ls -al
      - #Nothing
      - echo its me
  - name: window-2
    layout: even-vertical
    panes:
      - cat /etc/resolv.conf
      - #ignored
              
❯ tmass load two-window-per-session
tmass2015/01/14 15:45:21 failed to execute tmux send-keys -t two-window-per-session:0.0 ls -al : exit status 1 
 window not found: two-window-per-session:0

Do you have any idea what's wrong? I get the same error if I save my current setup and try to load it as well.

**Comment by fzerorubigd** [comment]
The exact same config works fine for me, Im on arch with exactly same version :/
```
uname -a
Linux elbix 3.17.6-1-ARCH #1 SMP PREEMPT Sun Dec 7 23:43:32 UTC 2014 x86_64 GNU/Linux
```
```
zsh --version
zsh 5.0.7 (x86_64-unknown-linux-gnu)
```

```
pacman -Q tmux  
tmux 1.9_a-1
```
I can not reproduce this problem in this configuration. could you please provide more data so I can regenerate this?

Thank you for using tmass!

**Comment by rif** [comment]
You are right, these two config options from my .tmux.conf caused the problem:
set -g base-index 1
set-window-option -g pane-base-index 1

It's working now, great tool. thanks!

**Comment by fzerorubigd** [comment]
Thank you. in fact this is one of my todo list :) 
https://github.com/fzerorubigd/tmass/blob/master/tmux/tmux.go#L122
So I keep this one open.
